langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -99,6 +116,16 @@ public isolated function sum(int... ns) returns int = @java:Method {
 
 # Returns the maximum of one or more int values.
 #
+# ```ballerina
+# int:max(50, 20, 30, 70, 65) ⇒ 70
+# 
+# int[] scores = [52, 45, 95, 76];
+# int:max(37, ...scores) ⇒ 95
+# 
```

Comment:
Shall we also add one like

```
[int, int, int] marks = [75, 80, 90];
int:max(...marks) ⇒ 90
```

May be applicable for other functions that have a mandatory first argument also.

Reply:
Could've been

```
int[3] marks = [75, 80, 90];
int:max(...marks) ⇒ 90
```

but this crashes atm - https://github.com/ballerina-platform/ballerina-lang/issues/39127.

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/38917/commits/57c2e044340a38b9a339ff102abd93730c3725d4